sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("30.0.0.0")]
-[assembly: AssemblyFileVersion("30.2.0.0")]
+[assembly: AssemblyVersion("30.1.0.0")]
```

Comment:
Update AssemblyVersion since this introduces a breaking change.

Reply:
upgrade the major version if there is a breaking change.

Reply:
I updated both major and minor version. 
AssemblyVersion changed from 30.0.0.0 tot 30.1.0.0
AssemblyFileVersion changed from 30.2.0.0 to 30.3.0.0